generators/app/templates/ext-command-js/vsc-extension-quickstart.md

```diff
@@ -14,12 +14,12 @@ We pass the function containing the implementation of the command as the second
 ## Get up and running straight away
 * press `F5` to open a new window with your extension loaded
 * run your command from the command palette by pressing (`Ctrl+Shift+P` or `Cmd+Shift+P` on Mac) and typing `Hello World`
-* set breakpoints in your code inside extension.ts to debug your extension
+* set breakpoints in your code inside extension.js to debug your extension
 * find output from your extension in the debug console
 
 ## Make changes
 * you can relaunch the extension from the debug toolbar after changing code in `extension.js`
-* you can also reload (`Ctrl+R` or `Cmd+R` on Mac) the VS Code window with your extension to load your changes
+* you can also reload (`Ctrl+Shift+F5` or `Cmd+Shift+F5` on Mac) the VS Code window with your extension to load your changes
```

Comment:
I found this binding from the debug toolbar. Cmd+R didn't work for me. From the toolbar, I can see that it is mapped to Cmd+Shift+F5